### PR TITLE
[WIP] improve translation redis

### PR DESCRIPTION
**Pull request by cordoval**
|Q            |A   |
|---          |--- |
|Fixed Tickets|#476|
                    

improve fetching redis translations in block

### CODE REVIEW

**Comment by mmoreram** [comment]
ping @cordoval is this ready?

File: src/Elcodi/Component/EntityTranslator/Services/CachedEntityTranslationProvider.php [hunk 1]
```diff
@@ -92,21 +92,19 @@ public function getTranslation(
         $cacheKey = $this->buildKey(
             $entityType,
             $entityId,
-            $entityField,
```
Comment (mmoreram) [inline src/Elcodi/Component/EntityTranslator/Services/CachedEntityTranslationProvider.php]: Why are you removing this?

**Comment by mmoreram** [comment]
ping2 @cordoval could you please rebase and add a small explanation (no more than 2 simple lines) about this improvement? Thanks :)

**Comment by cordoval** [comment]
no it is not ready, that is why it is a WIP, was expecting a direction or something

**Comment by mmoreram** [comment]
@cordoval Well, Is this change improving the old implementation? How?

**Pull request closed without merging**